graylog2-server/src/main/java/org/graylog2/rest/models/system/sessions/responses/DefaultSessionResponseFactory.java

```diff
@@ -48,7 +49,11 @@ public SessionResponse forSession(Session session) {
     }
 
     protected Date getValidUntil(Session session) {
-        return new DateTime(session.getLastAccessTime(), DateTimeZone.UTC).plus(session.getTimeout()).toDate();
+        if(session.getTimeout() < 0) { // means "session never expires", which is not possible in cookie-based auth
```

Comment:
Maybe add a space between if and `(`
```suggestion
        if (session.getTimeout() < 0) { // means "session never expires", which is not possible in cookie-based auth
```